pkg/v1/endpoints/books/request.go

```diff
@@ -0,0 +1,14 @@
+package books
+
+import (
+    "github.com/King-Rullmann-Book-Club/books-api/pkg/v1/service/books"
```

Comment:
Had a general question about this type of import. Does the repo need to be public for this to work? How does uncommitted/unmerged code (like here where the books service hasn't been made yet) get used? Why is this preferable to a relative path like (`import "books-api/pkg/v1/service/books"`)? Can non-github sources be used (e.g. gitlab).

Reply:
Not necessarily, this is just standard [go modules](https://go.dev/ref/mod#modules-overview). I've never tested out relative imports.